<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrizUI</class>
 <widget class="QMainWindow" name="MatrizUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>254</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>254</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>254</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>254</red>
        <green>254</green>
        <blue>254</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Matrizes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="OperacoesA">
      <item>
       <widget class="QPushButton" name="MultiplicacaoPorEscalarA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multiplicação
por escalar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PotenciacaoA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Potenciação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TranspostaA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Calcular 
Transposta</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="6" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="Igualdade">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Igualdade</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <layout class="QHBoxLayout" name="ConsultasA">
      <item>
       <widget class="QPushButton" name="TriangularA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Triangular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SimetricaA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Simétrica</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IdentidadeA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Identidade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PermutacaoA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Permutação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OrtogonalA">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ortogonal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="4">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Matriz A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="Encerrar">
      <property name="text">
       <string>Encerrar
Programa</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="6" colspan="3">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>135</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>135</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="4">
     <layout class="QVBoxLayout" name="Operacoes">
      <item>
       <widget class="QPushButton" name="Adicao">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans</family>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Subtracao">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans</family>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="Multiplicacao">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Lucida Sans</family>
          <pointsize>28</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="7" rowspan="2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nº de linhas</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="linhasB">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Nº de colunas</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="colunasB">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="SalvarB">
        <property name="text">
         <string>Salvar Matriz B</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="8">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Matriz B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>135</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="8">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="6">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>135</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="6" colspan="3">
     <layout class="QHBoxLayout" name="ConsultasB">
      <item>
       <widget class="QPushButton" name="TriangularB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Triangular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SimetricaB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Simétrica</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IdentidadeB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Identidade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PermutacaoB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Permutação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OrtogonalB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ortogonal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="9">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ATENÇÃO: Digite os elementos das matrizes nas respectivas tabelas das mesmas! Salve para realizar alguma operação!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="6" colspan="3">
     <layout class="QHBoxLayout" name="OperacoesB">
      <item>
       <widget class="QPushButton" name="MultiplicacaoPorEscalarB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Multiplicação
por escalar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PotenciacaoB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Potenciação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TranspostaB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Calcular 
Transposta</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTableWidget" name="MatrizA">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>401</width>
        <height>192</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="4" column="6" colspan="3">
     <widget class="QTableWidget" name="MatrizB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>401</width>
        <height>192</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Consultas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nº de linhas</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="linhasA">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Nº de colunas</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="colunasA">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="SalvarA">
        <property name="text">
         <string>Salvar Matriz A</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionCriar_novas_matrizes">
   <property name="text">
    <string>Criar novas matrizes</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
